doc/adrs/103_improve_feedback_messages.adoc

```diff
@@ -0,0 +1,50 @@
+= ADR-103 - Improves the existing feedback messages capability
+
+== Context
+
+We add an interface to return feedback messages to the UI, see link:./101_feedback_messages_on_actions.adoc[the adr] for more details.
+
+== Decision
+
+We have improved the possibility of returning a list of different messages, also on a success.
+We also have improved the representation of these messages to show them more clearly on the UI.
+
+=== Backend
+
+We changed the interface to :
```

Comment:
Minor, but there is no space before ":" (or ";", "!", "?") in english (its a french-specific rule). No need to make a new version just for that.